pulp_rpm/app/constants.py

```diff
@@ -1,5 +1,16 @@
 from types import SimpleNamespace
 
+
+RPM_PLUGIN_TYPES = SimpleNamespace(
+    PACKAGE='rpm.package',
+    ERRATA='rpm.update'
+)
+
+RPM_PLUGIN_TYPE_CHOICE_MAP = {
+    'packages': RPM_PLUGIN_TYPES.PACKAGE,
+    'errata': RPM_PLUGIN_TYPES.ERRATA
```

Comment:
Errata is a plural of Erratum, so if we use singular form for packages, we should use singular form for errata as well.